src/core/components/courses/TraineeTable.vue

```diff
@@ -135,7 +135,7 @@ export default {
       traineeValidations: {
         identity: { lastname: { required: requiredIf(this.addNewTraineeIdentityStep) } },
         local: { email: { required, email } },
-        contact: { phone: { frPhoneNumber } },
+        contact: { phone: { required, frPhoneNumber } },
```

Comment:
J'ai l'impression que sur traineeTable il y a un soucis avec la validation: 
- si l'entreprise n'est pas rempli j'ai le bon comportement
- si le nom n'est pas rempli j'ai une erreur
- si le numero de telephone n'est pas rempli, ca fonctionne bien. - Je pense qu'il manque une validation dans le back ici

Reply:
Je n'ai pas trouvé pourquoi le champ n'était pas required (dans sa validation il n'y a pas d'erreur au moment du touch alors que this.addNewTraineeIdentityStep est à true)

Reply:
Pour le requiredIf je me demande si ce n'est pas parce que requiredIf attend une fonction et qu'ici on donne un boolean. J'ai l'impression qu'a chaque fois qu'on l'utilise on le fait ainsi: 
`requiredIf(() => this.addNewTraineeIdentityStep)`